music/src/core/data.ts

```diff
@@ -200,6 +200,11 @@ export class DrumsConverter extends DataConverter {
         this.pitchToClass.set(p, c);
       });
     }
+    // We also add two extra pitches for the click-track.
```

Comment:
Why are these needed in the DataConverter?

Reply:
old attempts, sorry, removed.